### PR TITLE
Add update siteTittle in AtoM read-only feature

### PR DESCRIPTION
**Pull request by mamedin**
Connects to https://github.com/artefactual-labs/ansible-atom-replication/issues/6

The following 2 variables have been added:

* atom_replication_ro_sitetitle
* atom_replication_restore_sitetittle (default False)

When `atom_replication_restore_sitetittle` is set as True, the `atom_replication_ro_sitetitle` value is used as Read-Only AtoM Title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +145,8 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 | `atom_replication_synchronize_downloads_dir` | False | Boolean variable that enables the synchronization of the downloads directories |
 | `atom_replication_synchronize_uploads_dir` | False | Boolean variable that enables the synchronization of the uploads directories |
 | `atom_replication_ro_nginx_user` | `{% if ansible_os_family == 'RedHat' %}nginx{% elif ansible_os_family == 'Debian' %}www-data{% endif %}` | Nginx user on the AtoM read-only  server. It is needed to run the php symfony commands |
+| `atom_replication_restore_sitetitle` | False | Setting this variable as `True` the role will change the siteTitle on read-only, using the `atom_replication_ro_sitetitle` value |
```
Comment (amayita) [inline README.md:148]: When variable is set to `True` the role will change the siteTitle on the read-only bd, using the `atom_replication_ro_sitetitle` value

File: README.md [hunk 2]
```diff
@@ -145,6 +145,8 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
...
+| `atom_replication_ro_sitetitle` | Read Only Site | String that will be used as siteTitle in setting_i18n table when `atom_replication_restore_sitetitle=True` |
```
Comment (amayita) [inline README.md:149]: String used as siteTitle in the setting_i18n table when

**Review by amayita — CHANGES_REQUESTED** [review]
I tried to word this better, thanks!

File: README.md (hunk 2 above)
Comment (mamedin) [reply README.md]: Readme.md updated!

**Review by amayita — APPROVED** [review]
LGTM!

**Pull request merged**